app.vue

```diff
@@ -1,8 +1,28 @@
 <script setup lang="ts">
+const { changeLanguage, getLanguageIdFromCode, getAvailableLanguages } = useInternationalization();
 const { locale } = useI18n();
 const url = useRequestURL();
 const route = useRoute();
 
+const updateSessionWithLanguage = async () => {
+    const frontendLocale = locale.value;
+    await getAvailableLanguages();
+    await changeLanguage(getLanguageIdFromCode(frontendLocale));
+};
+
+await updateSessionWithLanguage();
+
+const { refreshCart } = useCart();
+const { getWishlistProducts } = useWishlist();
+const { refreshContext } = useCustomerStore();
+await refreshContext();
+
+useNotifications();
+useBreadcrumbs();
+
+refreshCart();
+getWishlistProducts();
```

Comment:
```suggestion
// only load wishlist products when they are not already loaded, the wishlist is enabled in the config and we are on the client so it doesn't block SSR
if (route.path !== '/wishlist' && wishlistEnabled && import.meta.client) {
    getWishlistProducts();
}
```

The first two checks are what we have in pond 0.1 currently and the last one I found in the shopware composables repo. I think they make sense, so it only loads the wishlist products when needed. What do you think?

Reply:
added!